src/main/java/mryapper/command/CommandList.java

```diff
@@ -0,0 +1,102 @@
+package mryapper.command;
+
+import mryapper.parser.DateTimeParser;
+import mryapper.task.Task;
+import mryapper.task.Deadline;
+import mryapper.task.Todo;
+import mryapper.task.Event;
+
+public class CommandList {
```

Comment:
Would it be better to split this class into multiple subclasses for each specific command?